src/bsoncxx/CMakeLists.txt

```diff
@@ -210,40 +197,46 @@ install(FILES
     COMPONENT dev
 )
 
-install(TARGETS
-    bsoncxx
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT runtime
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT dev
-)
-
 set(PACKAGE_INCLUDE_INSTALL_DIRS ${BSONCXX_HEADER_INSTALL_DIR})
 set(PACKAGE_LIBRARY_INSTALL_DIRS ${CMAKE_INSTALL_LIBDIR})
 set(PACKAGE_LIBRARIES bsoncxx)
 
 include(CMakePackageConfigHelpers)
 
-if (BUILD_SHARED_LIBS)
-    set(PKG "libbsoncxx")
-else()
-    set(PKG "libbsoncxx-static")
-endif()
+function(bsoncxx_install TARGET)
```

Comment:
Like with `bsoncxx_add_library`, here I'm making sure we do the same pattern for both installs via a CMake function

Reply:
Same comment as with `bsoncxx_add_library`.